views/job-roles.html

```diff
@@ -0,0 +1,32 @@
+{% extends "_includes/layouts/base.html" %}
+{% block content %}
+<main class="container mt-4">
+       
+    <section id="orders" class="">
+        <h2>Job Roles</h2>
+        {% if jobRoles and jobRoles.length >0 %}
+        <table class="table">
+            <thead>
+                <tr>
+                    <th scope="col">Job Title</th>
+                    <th scope="col">Status</th> 
+                    <th scope="col">Capability</th>                    
+                </tr>
+            </thead>
+            <tbody>
+                {% for job in jobRoles %}
+                <tr>
+                    <td>{{ job.jobRoleTitle }}</td>
+                    <td>Available</td>     
+                    <td>{{ job.capability_name }}</td>            
```

Comment:
This seems to not render any capability names

Reply:
It's working for me?